packages/manager/apps/web/client/app/domain/contact/dashboard/dashboard.html

```diff
@@ -0,0 +1,134 @@
+<div data-ui-view>
+    <h1 data-translate="domain_tab_CONTACT_title"></h1>
+    <div class="text-center" data-ng-if="$ctrl.loading">
+        <oui-spinner data-size="l"></oui-spinner>
+    </div>
+
+    <oui-message data-type="error" data-ng-if="$ctrl.saveError">
+        <span
+            class="d-block"
+            data-translate="domain_tab_CONTACT_description_error_message"
+        >
+        </span>
+        <small data-ng-bind-html="$ctrl.saveError"></small>
+    </oui-message>
+
+    <div data-ng-show="!$ctrl.loading">
+        <div class="row" data-ng-if="!$ctrl.zone">
+            <div class="col-md-12">
+                <div data-ovh-alert="{{alerts.main}}"></div>
```

Comment:
you are sure that alerts.main is defined

Reply:
nope, remove it indeed!